service-spec/service-spec-commons/src/main/java/io/hyscale/servicespec/commons/activity/ServiceSpecActivity.java

```diff
@@ -18,7 +18,13 @@
 import io.hyscale.commons.models.Activity;
 
 public enum ServiceSpecActivity implements Activity {
-    FAILED_TO_DESERIALIZE_REPLICAS("Cannot process the field 'replicas' with error {} , deployment continued with min 1 replica");
+    FAILED_TO_DESERIALIZE_REPLICAS("Cannot process the field 'replicas' with error {} , deployment continued with min 1 replica"),
+    SERVICE_NAME_MISMATCH("Service spec file convention to be <service-name>.hspec.yaml for better readability,But {} did not match with service name given in the service spec."),
```

Comment:
Expecting Service Spec file name to follow the convention <service-name>.hspec.yaml for readability. But ...
Follow the same format for profiles

Reply:
We can move service spec and profile file name format as a constant and use them instead

Reply:
Service name mismatch with hspec file name convention as <service-name>.hspec